packages/web/src/scss/components/Dropdown/_Dropdown.scss

```diff
@@ -1,5 +1,31 @@
+@use '../../tools/breakpoint';
+@use '../../tools/dictionaries';
 @use '../../tools/placement';
+@use 'theme';
 
 .Dropdown {
     @include placement.parent();
 }
+
+.Dropdown[class*='--alignment'] {
+    display: flex;
+}
```

Comment:
This relates to my question about nullable alignment: if you only define alignment for some breakpoints, display flex will be applied in all of them. Is it safe? 🤔